### PR TITLE
Bring in CSS [WIP]

### PR DESCRIPTION
**Pull request by blad**
@kaylah @emmabukacek

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-dist
-elm-stuff
+dist/
+elm-stuff/
```
Comment (emmabukacek) [inline app/.gitignore]: Haha, dang. Do these make a big difference?

File: app/elm-package.json [hunk 2]
```diff
@@ -6,11 +6,42 @@
...
+        "Material.Scheme",
+        "Material.Snackbar",
+        "Material.Spinner",
+        "Material.Slider",
+        "Material.Table",
+        "Material.Tooltip",
+        "Material.Toggles",
+        "Material.Tabs",
+        "Material.Typography",
+        "Material.Textfield"
```
Comment (emmabukacek) [inline app/elm-package.json]: Should we cut this down in scope now?

File: app/src/styles/GlobalCss.elm [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            [ display inlineBlock
+            , marginLeft (px 150)
+            , marginRight (px 80)
+            , verticalAlign middle
+            ]
+        , (#) BuyTickets
+            [ padding (px 16)
+            , paddingLeft (px 24)
+            , paddingRight (px 24)
+            , marginLeft (px 50)
```
Comment (emmabukacek) [inline app/src/styles/GlobalCss.elm]: Ooph; do we get `rems` and `ems` with elm-css? Not finding a quick list when I google for it.

File: app/src/styles/Stylesheets.elm [hunk 4]
```diff
@@ -0,0 +1,18 @@
+port module Stylesheets exposing (..)
```
Comment (emmabukacek) [inline app/src/styles/Stylesheets.elm]: Not sure we should expose the whole module here; perhaps it's worth narrowing down? Or is this what examples use?

File: elm-package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0",
+    "summary": "helpful summary of your project, less than 80 characters",
```
Comment (emmabukacek) [inline elm-package.json]: Can we go ahead and put in a valid summary here?

File: elm-package.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD3",
```
Comment (emmabukacek) [inline elm-package.json]: Probably should update these accordingly too.

File: app/.gitignore (hunk 1 above)
Comment (blad) [inline app/.gitignore]: I guess they really don't. Keeping the originals.

File: app/src/styles/GlobalCss.elm (hunk 3 above)
Comment (blad) [inline app/src/styles/GlobalCss.elm]: Looks like they are both available:
`rem` : https://github.com/rtfeldman/elm-css/blob/e4b4932f3c635030f3a550e6e113a16afd0808f6/src/Css.elm#L2343
`em`: https://github.com/rtfeldman/elm-css/blob/e4b4932f3c635030f3a550e6e113a16afd0808f6/src/Css.elm#L2292

File: app/elm-package.json (hunk 2 above)
Comment (blad) [inline app/elm-package.json]: Yeah. I don't our app needs to expose these. Removing.

File: app/src/styles/Stylesheets.elm (hunk 4 above)
Comment (blad) [inline app/src/styles/Stylesheets.elm]: This is how the examples are all set-up. This is likely how the `fileStructure` and  and `main` configs are read. Not 100% sure what `elm-css` needs exposed through the port though.

File: elm-package.json (hunk 5 above)
Comment (blad) [inline elm-package.json]: This whole file should be deleted actually. Consider it gone.

File: app/src/styles/GlobalCss.elm (hunk 3 above)
Comment (emmabukacek) [inline app/src/styles/GlobalCss.elm]: Way to just school with me with the source!

File: app/src/styles/Stylesheets.elm (hunk 4 above)
Comment (emmabukacek) [inline app/src/styles/Stylesheets.elm]: I suppose if the file is called "Stylesheets," it's probably fair.

**Pull request merged**